src/renderers/__tests__/ReactCompositeComponentState-test.js

```diff
@@ -226,13 +211,12 @@ describe('ReactCompositeComponent-state', () => {
       ['forceUpdate', 'blue'],
       // unmountComponent()
       // state is available within `componentWillUnmount()`
-      ['componentWillUnmount', 'blue'],
+      ['componentWillUnmount', 'blue']
```

Comment:
I don't know how I feel about this one. It seems odd to read it when so many functions will never have any more arguments. Especially for all the ones that only accept one argument. If I expected it to grow I'd use a "config" object instead.

This formatting seems odd:

```js
function foo(
  {
    x,
    y,
  },
) {}

foo({
  x,
  y,
});
```